content/en/agent/guide/datadog-disaster-recovery.md

```diff
@@ -0,0 +1,284 @@
+
+Each item can be added to the sync scope using the sync-cli configuration available in the documentation. Here's an example of a configuration file for syncing specific dashboards and monitors using name and tag filtering from an `EU` site to a `US5` site.
+
+```shell 
+destination_api_url="https://api.us5.datadoghq.com"
+destination_api_key="<US5_API_KEY>"
+destination_app_key="<US5_APP_KEY>"
+source_api_key="<EU_API_KEY>"
+source_app_key="<EU_APP_KEY>"
+source_api_url="https://api.datadoghq.eu"
+filter=["Type=Dashboards;Name=title","Type=Monitors;Name=tags;Value=sync:true"]
+
+# Make sure to increase the retry timeout to cope with the rate limit
+http_client_retry_timeout=600
+```
+
+Here's an example of a Sync-cli commands for syncing log configurations:
+
+```shell
+datadog-sync import –config config –resources users,roles,logs_pipelines,logs_pipelines_order,logs_indexes,logs_indexes_order,logs_metrics,logs_restriction_queries
+
+# remember to set the –cleanup=Force option
+datadog-sync sync –config config –resources users,roles,logs_pipelines,logs_pipelines_order,logs_indexes,logs_indexes_order,logs_metrics,logs_restriction_queries –cleanup=Force
+```
+
+<div class="alert alert-warning"> <strong>Sync-cli Limitation </strong><br><br>
+
+**Log Standard Attributes:** Sync-cli is regularly being updated with new resources. Currently, syncing Log standard attributes is not supported for private beta. If you use standard attributes with your log pipelines and are remapping your logs, attributes are a dependency that you need to manually re-configure in your secondary org. You can refer to the Datadog [standard attribute documentation][6] for support.
+</div>
+{{% /collapse-content %}}
+
+
+{{% collapse-content title=" 8. Verify availability at the secondary site" level="h5" %}}
+Verify that your secondary org is accessible and that your Dashboards and Monitors are copied from your primary org to your secondary org.
+{{% /collapse-content %}}
+
+
+{{% collapse-content title=" 9. Enable Remote Configuration [**RECOMMENDED]" level="h5" %}}
+[Remote configuration (RC)][7] is a Datadog capability that allows you to remotely configure and change the behavior of Datadog Agents deployed in your infrastructure. Remote Configuration is strongly recommended for a more seamless failover control. Alternatively, you can configure your Agents manually or using configuration management tools like Puppet, Ansible, Chef, etc. 
+
+Remote configuration will be turned on by default on your new organization and you can create new API keys that are RC-enabled by default for use with your Agent. See the documentation for [Remote configuration][7] for more information.
+{{% /collapse-content %}}
```

Comment:
I don't understand the action item here - Datadog recommends that we enable Remote Configuration, but it's enabled by default? What are we trying to get people to understand or do in this section?

Also, be careful about consistent capitalization 🙂 If you click through to the RC doc, you'll see that "Configuration" is always capitalized there, but it's not consistently capitalized here.